chainermn/communicators/_communication_utility.py

```diff
@@ -169,3 +171,15 @@ def chunked_bcast_obj(obj, mpi_comm, max_buf_len=256 * 1024 * 1024,
         obj = pickle.loads(pickled_bytes)
 
     return obj
+
+
+def _get_nccl_type_id(dtype):
+    if dtype == np.float16:
```

Comment:
Copy from pure_nccl_communicator.py ? The implementation must be unified

Reply:
Should I remove the `_get_nccl_type_id` in `pure_nccl_communicator.py`?

Reply:
Yes, please.